doc/host_create.md

```diff
@@ -195,26 +194,30 @@ Available keys for `--compute-attributes`:
 ```
 cpus          # number of CPUs
 memory        # string, amount of memory, value in bytes
+cpu_mode      # possible values: default, host-model, host-passthrough
 start         # Must be a 1 or 0, whether to start the machine or not
 ```
 
 Available keys for `--interface`:
 ```
-compute_type                      # one of [bridge, network]
-compute_network / compute_bridge  # name of interface according to type
-compute_model                     # one of [virtio, rtl8139, ne2k_pci, pcnet, e1000]
+type              # one of [bridge, network]
+network / bridge  # name of interface according to type
+model             # one of [virtio, rtl8139, ne2k_pci, pcnet, e1000]
```

Comment:
after the rebase, all the --interface attributes,  it should stay with "compute_" prefix.